docs/ss_slide_003a.md

```diff
@@ -1,22 +1,22 @@
 # Continuous CA frameworks Lenia and glaberish
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/glaberish/lenia_orbium.png)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/glaberish/lenia_orbium.png)
 
 Continuous CA dynamics are typically defined by computing a local context (neighborhood kernel convolution) followed by an update rule (growth function or genesis/persistence functions). The _Orbium_ rule in Lenia is shown above.
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/equations/lenia_annotated.png)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/equations/lenia_annotated.png)
 
 Lenia dynamics for the single and wide gliders from the _Scutium gravidus_ CA.
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/equations/glaberish_annotated.png)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/equations/glaberish_annotated.png)
 
 glaberish dynamics for the s613 dataset and 'frog' pattern.
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_002.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_003)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_002.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_003)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.